translator-gulp.js

```diff
@@ -5,13 +5,10 @@ var PluginError = gutil.PluginError;
 
 var Translator = require("./lib/translator.js");
 
-// consts
-const PLUGIN_NAME = 'gulp-translator';
+const PLUGIN_NAME = 'gulp-localizer';
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).